cloud_dashboard/src/service/utility.ts

```diff
@@ -277,20 +331,37 @@ export const readDataCache = async (entityColumnList: EntityColumn[]) => {
 export const convertEntityData = (
   rawDataList: EntityData[],
   entityColumnList: EntityColumn[],
-  cloudContext: CloudContext,
+  cloudContextList: CloudContext[],
   dataCache: { [key: string]: EntityData[] }) => {
   const newDataRecordList: DataRecord[] = [];
   for (const rawRecord of rawDataList) {
+    // convert (rawRecord: EntityData) => (dataRecord: DataRecord)
     const dataRecord: DataRecord = {
       id: rawRecord.id,
       value: {}
     };
+
+    // Read data column by launchTemplateColumn, convert as appropriate, and write it out.
     for (const launchTemplateColumn of entityColumnList) {
+      // Read data column by launchTemplateColumn.
       const rawValue = rawRecord.attributes[launchTemplateColumn.name];
-      dataRecord.value[launchTemplateColumn.name] = launchTemplateColumn.name === 'cloud_context'
-        ? cloudContext.labelName
-        : convertDataForUI(rawValue, launchTemplateColumn, dataCache);
+
+      // Convert as appropriate.
+      if (launchTemplateColumn.name !== 'cloud_context') {
+        // If it is not 'cloud_context', convert it by convertDataForUI().
+        dataRecord.value[launchTemplateColumn.name] = convertDataForUI(rawValue, launchTemplateColumn, dataCache);
+        continue;
+      }
+      // Search for possible candidates.
+      const temp = cloudContextList.filter((r) => {
```

Comment:
@RyoYamashita765 This can be:

```suggestion
      const cloudContexts = cloudContextList.filter((r) => {
```